clang/include/clang/3C/ConstraintVariables.h

```diff
@@ -574,7 +585,7 @@ class FunctionVariableConstraint : public ConstraintVariable {
   FunctionVariableConstraint()
       : ConstraintVariable(FunctionVariable, "", ""), FileName(""),
         Hasproto(false), Hasbody(false), IsStatic(false), Parent(nullptr),
-        IsFunctionPtr(false) {}
+        IsFunctionPtr(false), TypeParams(0) {}
```

Comment:
Looks like this was a bug waiting to happen, in the existing code (i.e., uninitialized field)?